src/tests.rs

```diff
@@ -1,15 +1,11 @@
-extern crate parking_lot;
+extern crate serial_test;
+use self::serial_test::serial;
```

Comment:
```suggestion

use serial_test::serial;
```

Do we actually need the `self`? If so, feel free to add it. I mostly wanted to opine about the spacing.